lib/ansible/plugins/inventory/aws_ec2.py

```diff
@@ -53,41 +53,41 @@
         strict_permissions:
           description: By default if a 403 (Forbidden) is encountered this plugin will fail. You can set strict_permissions to
               False in the inventory config file which will allow 403 errors to be gracefully skipped.
+    notes:
+      - Inventory files must end in '.aws_ec2.yml' or '.aws_ec2.yaml'
```

Comment:
redundant, this info is already in the description

Reply:
Fair, but notes are more obvious when reading the docs. I can remove this (I hadn't noticed it in the description when I added it)